ushahidi/platform-client
# Add first time login config screens

Also:
- Simplify modal logic: if visible is true, just open the modal.
- Default to close modal on overlay click
- Add option to show/hide modal close button
- Move appearance and general settings all into 1 directive

Test plan:
- Load the client
- Log in as admin 
First time config will popup
- Check various combinations of skipping, customizing etc.
To reset state: remove the 'first_login' flag from config table in db

Test general settings:
- Load /settings/general and check uploading header image works as expected.


[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/9)


Fixes ushahidi/platform#494
Fixes ushahidi/platform#677
